contracts/contracts/vault/VaultAdmin.sol

```diff
@@ -343,6 +343,45 @@ contract VaultAdmin is VaultStorage {
         emit AssetSupported(_asset);
     }
 
+    /**
+     * @notice Remove a supported asset from the Vault
+     * @param _asset Address of asset
+     */
+    function removeAsset(address _asset) external onlyGovernor {
+        require(assets[_asset].isSupported, "Asset not supported");
+        require(
+            IVault(address(this)).checkBalance(_asset) == 0,
```

Comment:
I'm wondering if we are actually going to be able to hit zero on some assets that are weird like stETH. Maybe this should be < 1e6 to allow for dust.

Reply:
Panda suggested to increase it to 1e13 (since the dust can exist across multiple strategies and it'd still be insignificant), so used that limit